code/modules/clothing/glasses/tajblind.dm

```
@@ -55,6 +64,30 @@
 			These current designs are adapted from recreations of the ancient veils, created by the Alchemists Guild. Technologically advanced and created to help Tajara adapt to life in the larger galactic community, they have systems built-in that allow them to have holographic huds, as well as corrective technology to help Tajaran overcome their genetic tritanopia colour blindness. <br>\
 			Availability on the wider market is highly restricted as a result of their cultural importance, as well as the patent held by the Alchemists Guild, and the lenses are very hard to reverse engineer. Popular theories suggest this as a result of the unique materials available on Adhomai, or the inability to recreate the light conditions of the Tajara homeworld."
 
+/obj/item/clothing/glasses/hud/tajblind/CtrlShiftClick(mob/user, modifiers)
+	if(!initial(hud_types))
+		to_chat(user, "<span class='notice'>You try to toggle electronics in [src] but there are none.</span>")
```

Can probably just let this fail silently
```suggestion
```